pkg/controller/route/routecontroller.go

```diff
@@ -95,15 +116,71 @@ func (rc *RouteController) reconcile(nodes []api.Node, routes []*cloudprovider.R
 			}
 			nameHint := string(node.UID)
 			wg.Add(1)
-			glog.Infof("Creating route for node %s %s with hint %s", node.Name, route.DestinationCIDR, nameHint)
-			go func(nodeName string, nameHint string, route *cloudprovider.Route, startTime time.Time) {
+			glog.V(2).Infof("Creating route for node %s %s with hint %s", node.Name, route.DestinationCIDR, nameHint)
```

Comment:
please revert to glog.Infof - this is pretty useful log (and there shouldn't be many of those)